core/spec/models/spree/concerns/user_address_book_spec.rb

```diff
@@ -167,9 +244,22 @@ module Spree
           expect(subject).to eq address
         end
 
-        it "sets it as default" do
-          subject
-          expect(user.default_address).to eq address
+        context "sets it as default" do
+          context "shipping address" do
+            it do
+              subject
+              expect(user.default_address).to eq address
```

Comment:
I think you can use this syntax: `it { is_expected.to eq address }`

Reply:
Not in this case, as I'm not doing an `expect(subject).to`, but I did it in the following one